configs/config.toml.SAMPLE

```diff
@@ -1,106 +1,95 @@
+# used together or separately.
+#
+# RequestsPerSecondLimit sets the maximum average number of requests
+# per second which can be executed against the Ethereum node.
+# All types of Ethereum node requests are rate-limited,
+# including view function calls.
+#
+# RequestsPerSecondLimit = 150
+
+# ConcurrencyLimit sets the maximum number of concurrent requests which
+# can be executed against the Ethereum node at the same time.
+# This limit affects all types of Ethereum node requests,
+# including view function calls.
+#
+# ConcurrencyLimit = 30
+
+# BalanceAlertThreshold defines a minimum value of the operator's account
+# balance below which the client will start reporting errors in logs.
+# A value can be provided in `wei`, `Gwei` or `ether`, e.g. `7.5 ether`,
+# `7500000000 Gwei`.
+#
+# BalanceAlertThreshold = "0.5 ether" # 0.5 ether (default value)
 
-# Uncomment to enable the metrics module which collects and exposes information
-# useful for external monitoring tools usually operating on time series data.
+[network]
+Peers = [
+	"/ip4/127.0.0.1/tcp/3919/ipfs/16Uiu2HAmFRJtCWfdXhZEZHWb4tUpH1QMMgzH1oiamCfUuK6NgqWX",
+]
+Port = 3920
+
+# Uncomment to override the node's default addresses announced in the network
+# AnnouncedAddresses = ["/dns4/example.com/tcp/3919", "/ip4/80.70.60.50/tcp/3919"]
+
+# Uncomment to enable courtesy message dissemination for topics this node is
+# not subscribed to. Messages will be forwarded to peers for the duration
+# specified as a value in seconds.
+# Message dissemination is disabled by default and should be enabled only
+# on selected bootstrap nodes. It is not a good idea to enable dissemination
+# on non-bootstrap node as it may clutter communication and eventually lead
+# to blacklisting the node. The maximum allowed value is 90 seconds.
+#
+# DisseminationTime = 90
+
+[storage]
+DataDir = "/my/secure/location"
+
+# Metrics collects and exposes information useful for external monitoring tools usually
+# operating on time series data.
 # All values exposed by metrics module are quantifiable or countable.
 #
 # The following metrics are available:
 # - connected peers count
 # - connected bootstraps count
 # - eth client connectivity status
-#
-# The port on which the `/metrics` endpoint will be available and the frequency
-# with which the metrics will be collected can be customized using the
-# below parameters.
-# [Metrics]
-    # Port = 8080
-    # NetworkMetricsTick = 60
-    # EthereumMetricsTick = 600
+[metrics]
+Port = 8080
+# NetworkMetricsTick = 60
+# EthereumMetricsTick = 600
 
-# Uncomment to enable the diagnostics module which exposes information useful
-# for debugging and diagnostic client's status.
-#
 # Diagnostics module exposes the following information:
 # - list of connected peers along with their network id and ethereum operator address
 # - information about the client's network id and ethereum operator address
+[Diagnostics]
```

Comment:
Should we lowercase `[Diagnostics]`?

Reply:
8b94282783a8419d93280ed05ee8615fe468804f